includes/admin/reporting/reports.php

```diff
@@ -2398,33 +2275,17 @@ function edd_register_customer_report( $reports ) {
 			),
 		) );
 
-		$reports->register_endpoint( 'lifetime_value_of_customer', array(
-			'label' => __( 'Average Lifetime Value', 'easy-digital-downloads' ),
-			'views' => array(
-				'tile' => array(
-					'data_callback' => function () use ( $exclude_taxes ) {
-						$stats = new EDD\Stats();
-						return $stats->get_customer_lifetime_value( array(
-							'function'      => 'AVG',
-							'exclude_taxes' => $exclude_taxes,
-							'output'        => 'formatted',
-						) );
-					},
-				),
-			),
-		) );
-
-		$reports->register_endpoint( 'average_customer_value', array(
-			'label' => __( 'Average Value', 'easy-digital-downloads' ),
+		$reports->register_endpoint( 'average_revenue_per_customer', array(
+			'label' => __( 'Average Revenue per Customer', 'easy-digital-downloads' ),
```

Comment:
I feel like the triangle on this tile should be colored the same way the triangles are on the overview tiles...right now it's just gray.